src/pydata_sphinx_theme/theme/pydata_sphinx_theme/components/search-field.html

```diff
@@ -6,7 +6,6 @@
   <input type="search"
          class="form-control"
          name="q"
-         id="search-input"
```

Comment:
It's misleading to put an id on this element because this template can currently be rendered on the same page more than once. This definitely happens in the PST docs, for example, on the search results page.